### PR TITLE
Flyer Carousel

### PR DESCRIPTION
**Pull request by lucasstranger1**
### 1. What type of PR is this? (check all applicable)
- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

### 2. Description

This PR adds flyer carousel section on the events page.

### 5. Add tests?
- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### 6. Remarks, notes, or any future suggestions?
Let me know if there's anything that we need to adjust!

### 7. Add to the documentation(s)?
- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 docs
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: files/about.html [hunk 1]
```diff
@@ -384,7 +384,7 @@ <h2 class="team-member-name">
                                 </h2>
 
                                 <h3 class="team-member-name">
-                                    
+                                    Michell Argus
```
Comment (r0m3c) [inline files/about.html:387]: Hey Lucas, for Michell's information section, let's create a separate pr for her so we can merge that as soon as possible. @lucasstranger1

Comment (r0m3c) [reply files/about.html:387]: Let's keep this pr related to the carousel only. Thanks in advance

**Review by r0m3c — CHANGES_REQUESTED** [review]
The carousel looks great as a first draft. The functionality is great and looks visually appealing. Here are some changes that I noticed were slightly different from the original website:

1) Footer looks shrunk. Let's try to fix the spacing to keep it the same.
Screenshot 2024-10-22 at 12 28 08 PM

2) The font of all letters on the website look different than the original website. Let's try to keep the font the same. Also, the left side (where logo and "John Jay Computer Science Society Club" looks a bit uncentered.
Screenshot 2024-10-22 at 12 28 25 PM

3) The spacing of this looks a bit larger than the original section and the header isn't bold as well. If we can keep the spacing similar to the original spacing and the font the same, we should be good to go.
Screenshot 2024-10-22 at 12 28 37 PM
Screenshot 2024-10-22 at 12 38 21 PM

4) In a small screen the carousel isn't able to go to the next flyer. If we can shrink the carousel for small screens that would be great, also without affecting the spacing of the website.
Screenshot 2024-10-22 at 12 29 49 PM

Overall, great job. In our CSS code let's try to make custom selectors that only affect the carousel code rather than having it affect other elements (like the header, footer, fonts, ect.) in our html code. Let me know if you have any clarifications, questions, or assistance. Happy to help.

- Leandro Gamarra

File: files/about.html (hunk 1 above)
Comment (lucasstranger1) [reply files/about.html]: I just saw the comment.
I will try to create another branch that only changes carousel features. I think I accidentally touched the original CSS code.

Comment (r0m3c) [reply files/about.html]: Sounds good

**Pull request closed without merging**